### DISCUSSION TITLE
Nuxt-security is not working with nitro preset vercel-edge

### DISCUSSION THREAD

**Discussion by RadoSabo** [Baroshem/nuxt-security#184] [Q&A] [answered]
Hello, 

do you have any solution how to unlock vercel deployment of nuxt project with nuxt-security running on the edge please?

This is how it looks when I add the module into the clean project and deploy it on the vercel with environment variable NITRO_PRESET='vercel-edge'
https://nuxt3-test-rho.vercel.app/

**Comment by RadoSabo** [comment, marked as answer]
fixed by disabling rateLimiter
https://github.com/Baroshem/nuxt-security/issues/137

**Comment by Baroshem** [comment]
Hey, I am glad that you have found a solution.

Built in rate limiter is unfortunately not suitable for production application.

**Reply by Baroshem** [reply]
Could you add some comments in this discussion? https://github.com/Baroshem/nuxt-security/discussions/140

I think that rate limiter is no longer usable and causes more problems than it solves.
